src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -403,9 +409,12 @@ public void onTraversalStart() {
         if (RSCRIPT_FILE != null) {
             rScriptExecutor = new RScriptExecutor();
             if(!rScriptExecutor.externalExecutableExists()) {
-                Utils.warnUser(logger, String.format(
+                if(!disableRScriptExecutor) {
+                    throw new UserException("Rscript not found in environment path. Fix executor or run with disableRScriptExecutor argument to generate Rscript without running.");
```

Comment:
If you are adding an argument "--<cli argument>" so its easy for the user to paste it into their command line.

Reply:
Also be sure that the argument is properly kebab-cased (`--disable-rscriptexecutor`) in the message.